extensions/amp-story/1.0/amp-story.css

```diff
@@ -275,6 +275,10 @@ amp-story-page[distance="2"] {
   transform: translateY(100%) !important;
 }
 
+.i-amphtml-story-desktop-one-panel amp-story-page[distance="2"] {
+  transform: translateY(100vh) !important;
```

Comment:
I'm wondering if we could just change it to 100vh all the time.
The historical context behind the 100% is that it's hard to trust VH because of browser expanded/minified UIs on user scroll. 100vh is not always 100%.

If you know the answer to this question (always using 100vh) and feel confident about it, plz do it. If not sure, let's keep it this way and simply move the override to the onepanel CSS file.

Reply:
AFAIK it should be fine, since `100vh >= 100%`, so it should still be off-screen.

Reply:
> AFAIK it should be fine, since 100vh >= 100%, so it should still be off-screen.

Agreed that in this case we can use 100vh since it will always be off screen. Lets go that route.